lmdeploy/turbomind/deploy/converter.py

```diff
@@ -161,17 +161,77 @@ def pack_model_repository(workspace_path: str):
                dst=osp.join(model_repo_dir, 'postprocessing'))
 
 
+def find_quantization_config(nested, target_key):
```

Comment:
Will this recurse infinitely if the config back links to itself?

Reply:
The recursive procedure happens only when the `nested` is a dict or a list. 
I don't think there is a case that the config links back to itself given it is a dict or a list.